enterprise/j2ee.dd.webservice/src/org/netbeans/modules/j2ee/dd/impl/resources/XMLSchema.dtd

```diff
@@ -0,0 +1,402 @@
+<!-- DTD for XML Schemas: Part 1: Structures
```

Comment:
It would be great if you could add the source and license information (see `xml.xsd` for a sample) here too. This is a file from W3C.